lib/lightning/workflows/snapshot.ex

```diff
@@ -52,6 +52,11 @@ defmodule Lightning.Workflows.Snapshot do
       field :cron_expression, :string
       field :enabled, :boolean
       field :type, Ecto.Enum, values: [:webhook, :cron]
+      field :has_auth_method, :boolean, virtual: true
```

Comment:
@elias-ba I'm not quite able to work out how this is used, I see it's in the workflow_params, but I can't find anything that sets it?

Reply:
Because we are using a snapshot to build the changeset in the workflow Canvas, when viewing a workflow that has auth methods set to its trigger, the UI would break without this.

Reply:
I guess what I'm saying is, I can't see anything that sets it... so affect it will always be `nil`. If it's not ever being set we should make an effort to remove it in another PR perhaps?